assets/js/components/DashboardViewIndicator.js

```diff
@@ -35,8 +36,15 @@ const DashboardViewIndicator = () => {
 	const isGA4DashboardView = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).isGA4DashboardView()
 	);
+	const analyticsModuleConnected = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleConnected( 'analytics' )
+	);
```

Comment:
We should avoid rendering this component at all if Analytics isn't connected. This is what `whenActive()` is useful for :)